client/sites/settings/administration/tools/reset-site/index.jsx

```diff
@@ -264,6 +247,15 @@ function SiteResetCard( {
 						</>
 					) }
 					<hr />
+					{ ! isAtomic && (
+						<ExportNotice
+							siteSlug={ selectedSiteSlug }
+							siteId={ siteId }
+							warningText={ translate(
+								'Before resetting your site, consider exporting its content as a backup'
```

Comment:
We could be a bit more direct in the copy here. Something like:

> Export a backup of your content before resetting this site. [Export content]

Reply:
👍 

I've added Needs Copy Review label to loop in the pros.